src/app/[locale]/(auth)/login/_components/login-form.tsx

```
@@ -0,0 +1,87 @@
+"use client";
+
+import { LoginFields, useLoginSchema } from "@lib/schemas/auth.schema";
+import React from "react";
+import { SubmitHandler, useForm } from "react-hook-form";
+import {zodResolver} from '@hookform/resolvers/zod'
+import { Form, FormControl, FormField, FormItem, FormMessage } from "@components/ui/form";
```

ESLint

[error] Missing semicolon.

(semi)